### PR TITLE
Adding rust client information to docs

### PR DESCRIPTION
**Pull request by hayleycd**
#### Summary
Partially addresses #324 by adding information on the Rust Client. 

#### Release Note
None

### CODE REVIEW

**Comment by hayleycd** [comment]
@flavio Would you please review? CC @haydentherapper @cmurphy

File: content/en/language_clients/rust/overview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+type: docs
+category: Rust
+title: Rust Client Overview
+weight: 5
+---
+
+[`sigstore`](https://crates.io/crates/sigstore) is a crate designed to interact with Sigstore architecture.
+
+This crate is under active development, but will not be considered stable until the 1.0 release.
```
Comment (haydentherapper) [inline content/en/language_clients/rust/overview.md:10]: Can we include a link to https://github.com/sigstore/sigstore-rs/issues/274 to track the 1.0 release?

File: content/en/language_clients/rust/overview.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Features
+
+- Cosign sign and verify
```
Comment (haydentherapper) [inline content/en/language_clients/rust/overview.md:14]:
```suggestion
- Container and binary signing and verification
```

Comment (haydentherapper) [reply content/en/language_clients/rust/overview.md:14]: To distinguish from Cosign, the CLI

File: content/en/language_clients/rust/overview.md (hunk 1 above)
Comment (haydentherapper) [inline content/en/language_clients/rust/overview.md:10]: Can we bold this to emphasize it's not stable and needs more changes to be conformant to other Sigstore libraries?

Comment (hayleycd) [reply content/en/language_clients/rust/overview.md:10]: Good call. Updating now.

**Comment by hayleycd** [comment]
@haydentherapper comments have been addressed. Thank you for the review.

**Review by haydentherapper — APPROVED** [review]
Going to merge now, if Flavio has any follow up comments, we can address them in another PR. Thanks!

**Comment by flavio** [comment]
Sorry for the late response. Everything LGTM. Thanks for having updated the docs to mention Rust :)

**Pull request merged**